src/test/java/automationpractice/com/pages/HomePage.java

```diff
@@ -4,15 +4,27 @@
 import net.serenitybdd.core.pages.WebElementFacade;
 import org.openqa.selenium.support.FindBy;
 
+import javax.swing.*;
+
 public class HomePage extends PageObject
 {
 
     @FindBy(id = "create_account_error")
     private WebElementFacade homePageError;
 
+    @FindBy(xpath = "//*[contains(@class,'replace-2x img-responsive') and contains(@title,'Faded Short Sleeve T-shirts')]")
+    private WebElementFacade clickOnNewProductImage;
+    @FindBy(xpath = "//*[contains(@class,'alert-success')]")
+    private WebElementFacade SuccessfulMessage;
+
```

Comment:
SuccessfulMessage z małej litery. 
clickOnNewProductImage - popraw ten xpath

Reply:
clickOnNewProductImage - popraw xpath. contains (@class, 'ajax_block_product col-xs-12 col-sm-4 col-md-3 first-in-line first') - jesli zmieni się chociaż kolejność tych klas, to to już nie przejdzie

Reply:
clickOnNewProductImage - popraw xpath. contains (@Class, 'ajax_block_product col-xs-12 col-sm-4 col-md-3 first-in-line first') - jesli zmieni się chociaż kolejność tych klas, to to już nie przejdzie